src/i18n/swg-strings.js

```diff
@@ -89,4 +89,68 @@ export const SWG_I18N_STRINGS = {
     'zh-hk': '透過 Google 提供內容',
     'zh-tw': '透過 Google 捐款',
   },
+  'REGWALL_REGISTERED_BEFORE_LANG_MAP': {
```

Comment:
"ALREADY_REGISTERED" seems a little more natural to me than "REGISTERED_BEFORE", partially because I've been referencing API best practices lately: https://google.aip.dev/140#prepositions

Reply:
Done.